### DISCUSSION TITLE
Can I use AutoModelForSequenceClassification?

### DISCUSSION THREAD

**Discussion by cristobalmackenzie** [axolotl-ai-cloud/axolotl#1721] [Q&A] [answered]
Hi ! I've been looking at examples and I'm not sure if I can use AutoModelForSequenceClassification, and if so, how exactly to format the dataset. I'm assuming I can use a "text" and "label" dataset like in HuggingFace examples, but I'm not sure if this is supported.

Only mention of SequenceClassification I've found so far is [this](https://github.com/OpenAccess-AI-Collective/axolotl/pull/773#issue-1956041206).

Thanks!

**Comment by NanoCode012** [comment, marked as answer]
Hey, I apologize for the late message. I'm answering this for any future readers.

The current code base is written around `AutoModelForCausalLM`. Unfortunately, we still don't have support for `AutoModelForSequenceClassification`.
